packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -89,7 +91,32 @@ export async function testApi(event: AWSLambda.APIGatewayProxyEvent): Promise<AW
   const parameters = JSON.parse(event.body).parameters;
   const endpointId = event.pathParameters.endpointId;
 
-  const [err, result] = await handlers.testApi(parsedConfig, endpointId, parameters);
+  const [err, result] = await handlers.processHttpRequest(parsedConfig, endpointId, parameters);
+  if (err) {
+    return { statusCode: 400, body: JSON.stringify({ error: err.toString() }) };
```

Comment:
Is 400 (bad request) the right status code? This implies a request/client error

Reply:
Those error codes need to be revisited for sure. There's already an issue for that https://api3dao.atlassian.net/browse/AN-506

Reply:
Happy with that then. I've added an explicit comment to the ticket about reviewing the status codes as well